v0/destinations/ga4/data/GA4PageConfig.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "destKey": "path",
+    "sourceKeys": "context.page.path",
+    "required": false
+  },
+  {
+    "destKey": "referrer",
+    "sourceKeys": "context.page.referrer",
+    "required": false
+  },
+  {
+    "destKey": "search",
+    "sourceKeys": "context.page.search",
+    "required": false
+  },
+  {
+    "destKey": "title",
+    "sourceKeys": "context.page.title",
+    "required": false
+  },
+  {
+    "destKey": "url",
+    "sourceKeys": "context.page.url",
+    "required": false
+  }
+]
```

Comment:
We can get all of these from properties, why we are not taking it from there?

Reply:
Ideally @akashrpo lets reference properties first then contextual data (customers can customize the properties values)

Reply:
parameters passed in properties will override these values as we are taking custom values as well

Reply:
Got it make sense